sources/api/migration/migrator/src/args.rs

```diff
@@ -114,15 +117,36 @@ impl Args {
                     migrate_to_version = Some(br.version_id)
                 }
 
-                _ => usage(),
+                "--root-path" => {
+                    let path_str = iter
+                        .next()
+                        .unwrap_or_else(|| usage_msg("Did not give argument to --root-path"));
+                    trace!("Given --root-path: {}", path_str);
+                    root_path = Some(PathBuf::from(path_str));
+                }
+
+                "--metadata-directory" => {
+                    let path_str = iter.next().unwrap_or_else(|| {
+                        usage_msg("Did not give argument to --metadata-directory")
+                    });
+                    trace!("Given --metadata-directory: {}", path_str);
+                    metadata_path = Some(PathBuf::from(path_str));
+                }
+                _ => usage_msg(format!("Unable to parse input '{}'", arg)),
             }
         }
 
         Self {
-            datastore_path: datastore_path.unwrap_or_else(|| usage()),
+            datastore_path: datastore_path
+                .unwrap_or_else(|| usage_msg("--datastore_path is empty")),
```

Comment:
```suggestion
                .unwrap_or_else(|| usage_msg("--datastore-path must be specified")),
```
Kind of sounded like it was specified but given an empty argument.

(Note also the underscore -> hyphen)

Same below.

Reply:
Addressed by:

> webern force-pushed the webern:pr/signed-migrations branch from f481a2d to 4443441